flush_loader.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+#USB=/dev/ttyUSB0
+USB=/dev/ttyUSB1
+BL2_EL3=1
+
+case "$1" in
+"hikey")
+	echo "Download recovery images into HiKey platform"
+	if [ $BL2_EL3 ]; then
+		echo "Flush recovery.bin into eMMC"
```

Comment:
Is this reversed? `recovery` should be when `BL2_EL3=0`?

Reply:
When $BL2_EL3 is active, we have recovery.bin.

Reply:
Shouldn't it be the other way round? When $BL2_EL3 is active, then there's no need for BL1, i.e. no need for recovery?

Reply:
Recovery mode is always at there. We need to recovery from brick state.

When $BL2_EL3 is active, we have three binaries output. They're l-loader.bin, fip.bin and recovery.bin.
When $BL2_EL3 is inactive, we have two binaries output. They're l-loader.bin and fip.bin. We need to use l-loader.bin in recovery mode.

Reply:
Ah, got it. Thanks!

Reply:
Yes, it's just a demo. Since we'll migrate to BL2_EL3, we always use it with $BL2_EL3 active. If user wants to use it for old images, he could update the script file by himself.

I don't want to provide too much parameters to the script file. There's only one parameter -- platform.

Reply:
Ok. Understood. Maybe we can consider removing the "$BL2_EL3 is not active" part to avoid confusion or flashing error by regular users, since all the other (make)files don't support BL2_EL3=0 anyway.

Reply:
Actually never mind. The 'not active' part is good to highlight the difference in flashing. Can be useful reference.